modules/util/css-builder/build.gradle

```diff
@@ -1,10 +1,21 @@
 import com.liferay.gradle.util.copy.StripPathSegmentsAction
 
 apply plugin: "com.liferay.maven.plugin.builder"
+apply plugin: "maven-publish"
 
 task copyPortalCommonCSSJar(type: Copy)
 task unzipPortalCommonCSS(type: Copy)
 
+def repoPath = new File('./src/test/resources/com/liferay/css/builder/maven/repo/').getAbsolutePath().toString()
```

Comment:
The repoPath should not go into the source folder, but instead should be pointing to  test-output path.